NEWS.md

```diff
@@ -1,3 +1,36 @@
+# admiral 0.12.0.9000
```

Comment:
> Update `NEWS.md` under the header `# admiral (development version)`

The news file doesn't use the dev version numbers: just keep it as  `# admiral (development version)`